src/utils/packageUtils.ts

```diff
@@ -167,20 +192,42 @@ export function escapeInstallationKey(key?: string): Nullable<string> {
 }
 
 /**
- * Given 0Ho the package type type (Managed, Unlocked, Locked(deprecated?))
+ * Fetch the PackageType for a given package version ID
```

Comment:
Do you think this could fit as a static method on the `Package` class or does the `04t` ID cause  it to not make sense?

Reply:
I would like to see this go away.

We should expose getter on package2 for this field

We should expose getter on package version for package type

Not sure if that work is out of scope.